data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/AbstractLogstashPluginAttributesMapperTest.java

```diff
@@ -6,27 +6,31 @@
 package org.opensearch.dataprepper.logstash.mapping;
```

Comment:
Please also add a test case that verifies that this works when the attribute name is mapped to a different name.